ambry-server/src/test/java/com.github.ambry.server/AmbryRequestsTest.java

```diff
@@ -616,6 +662,87 @@ private Response sendRequestGetResponse(RequestOrResponse request, ServerErrorCo
     return response;
   }
 
+  /**
+   * Sends and verifies that an operation specific request works correctly.
+   * @param request the {@link Request} to send to {@link AmbryRequests}
+   * @param expectedErrorCode the {@link ServerErrorCode} expected in the response. For some requests this is the
+   *                          response in the constituents rather than the actual response ({@link GetResponse} and
+   *                          {@link ReplicaMetadataResponse}).
+   * @param forceCheckOpReceived if {@code true}, checks the operation received at the {@link Store} even if
+   *                             there is an error expected. Always checks op received if {@code expectedErrorCode} is
+   *                             {@link ServerErrorCode#No_Error}. Skips the check otherwise.
+   * @throws InterruptedException
+   * @throws IOException
+   */
+  private void sendAndVerifyOperationRequest(RequestOrResponse request, ServerErrorCode expectedErrorCode,
+      Boolean forceCheckOpReceived) throws InterruptedException, IOException {
+    storageManager.resetStore();
+    RequestOrResponseType requestType = request.getRequestType();
+    Response response = sendRequestGetResponse(request,
+        requestType == RequestOrResponseType.GetRequest || requestType == RequestOrResponseType.ReplicaMetadataRequest
```

Comment:
How about using following code to make this statement slightly more readable?
```
    Response response = sendRequestGetResponse(request,
        EnumSet.of(RequestOrResponseType.GetRequest, RequestOrResponseType.ReplicaMetadataRequest).contains(requestType)
            ? ServerErrorCode.No_Error : expectedErrorCode);
```

Reply:
Done